econml/tests/test_bootstrap.py

```diff
@@ -150,3 +150,26 @@ def test_backends(self):
                 lower, upper = bs.predict_interval(x, lower=10, upper=90)
                 for bound in [lower, upper]:
                     self.assertEqual(np.shape(est.predict(x)), np.shape(bound))
+
```

Comment:
Can you add some more tests for other estimators?

Reply:
Yes, that's a good idea.  Will do.